src/Microsoft.HttpRepl/OpenApi/SwaggerV1EndpointMetadataReader.cs

```diff
@@ -24,9 +24,9 @@ public IEnumerable<EndpointMetadata> ReadMetadata(JObject document)
                 globalConsumes = new JArray();
             }
 
-            if (document["apis"] is JObject obj)
+            if (document["apis"] is JArray jArray)
             {
-                foreach (JProperty property in obj.Properties())
+                foreach (JObject obj in jArray)
```

Comment:
I mostly modified this based on example I found here:  https://github.com/swagger-api/swagger-codegen/blob/4607a90d7b69463a0ae8fc94fac68fc95d80965e/modules/swagger-codegen/src/test/resources/1_2/petstore-1.2/pet. I don't have a lot of context on what the schema should look like. Let me know your thoughts

Reply:
Seems like the best way would be to create a site with a Swagger v1 endpoint and try it out, no?  You can validate the expected schema and get some manual validation of your changes at the same time.

Reply:
I spent sometime on this today. I couldn't find a good documentation on how to do this. I'll look into it again over the weekend.

Reply:
If you're not confident on it, can we file an issue for follow-up/further investigation?  And then I'm fine with either taking this part of the change (because that's what the example indicates it should be) or reverting it (because keeping the original is a safe play).

Reply:
Submitted new issue here: https://github.com/aspnet/HttpRepl/issues/92 for further investigation. Thanks!